### PR TITLE
updates to more correctly obey the style guide

### PR DESCRIPTION
**Pull request by heckj**
Ref issue: #7030

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/7213)

### CODE REVIEW

**Comment by heckj** [comment]
@steveperry-53 does this do what you’re after in terms of more correctly obeying the style guide?

File: docs/user-journeys/users/cluster-operator/foundational.md [hunk 1]
```diff
@@ -10,8 +10,8 @@ track: "USERS › CLUSTER OPERATOR › FOUNDATIONAL"
 
 {% capture overview %}
 
-If you want to learn how to get started managing and operating a Kubernetes cluster, this page and the linked topics will introduce you to foundational concepts and tasks.
-The concepts introduce you to a Kubernetes cluster and key concepts to understand and manage it, favoring focusing on the cluster itself over the software running within the cluster.
+If you want to learn how to get started managing and operating a Kubernetes cluster, this page and the linked topics introduces you to foundational concepts and tasks.
```
Comment (tengqm) [inline docs/user-journeys/users/cluster-operator/foundational.md]: introduces -> introduce?

**Comment by heckj** [comment]
@tengqm thanks, good catch - definitely more correct. Updated

**Comment by steveperry-53** [comment]
@heckj Can you reopen this? I don't understand why it got merged. Thanks.

File: docs/user-journeys/users/cluster-operator/foundational.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ Metrics
 
 ## Get an overview of Kubernetes
 
-If you have not already done so, start your understanding by reading through [What is Kubernetes?](/docs/concepts/overview/what-is-kubernetes/), which will introduce you to a number of basic concepts and terms.
+If you have not already done so, start your understanding by reading through [What is Kubernetes?](/docs/concepts/overview/what-is-kubernetes/), which introduces a number of basic concepts and terms.
 
 Kubernetes is quite flexible, and can a cluster can be run in a wide variety of places. You can interact with Kubernetes entirely on your own laptop or local development machine with it running within a virtual machine. Kubernetes can also run on virtual machines hosted either locally or in a cloud provider, and you can run a kubernetes cluster on bare metal.
```
Comment (steveperry-53) [inline docs/user-journeys/users/cluster-operator/foundational.md]: and can a cluster can -> and a cluster can

**Comment by steveperry-53** [comment]
I know this is a weird one, but I think "If there are more than one node in your cluster" should be "If there is more than one node in your cluster".

**Comment by steveperry-53** [comment]
each of the nodes are connected -> the nodes are connected

**Comment by steveperry-53** [comment]
Regardless of how many nodes, all Kubernetes clusters will generally have -> Regardless of the number of nodes, all Kubernetes clusters generally have

**Comment by steveperry-53** [comment]
a Kubernetes cluster is by setting them up -> a Kubernetes cluster is by setting it up

Alternative: setting one up

**Comment by steveperry-53** [comment]
Are you sure that the Katacoda Playground uses Minikube?

**Comment by steveperry-53** [comment]
to play with in on the web -> to play with on the web

**Comment by steveperry-53** [comment]
"Be familiar with how to Organizing Cluster Access" doesn't work. Maybe this: Be familiar with Organizing Cluster Access by using configuration files.

**Comment by steveperry-53** [comment]
Need a period after, "these resources at Understanding Kubernetes Objects".

**Comment by steveperry-53** [comment]
Our style guide has a section named **Use simple and direct language**. One of the examples we give is to say "To" instead of "In order to". Personally, I would like to see another example added to this section in the guide: Use "in" instead of "within". Let's see what the others think.

So I would prefer to see "in the Kubernetes documentation" rather than "within" the Kubernetes documentation. I would prefer "in the cluster" rather than "within" the cluster.

**Comment by steveperry-53** [comment]
This sentence needs a couple of commas.

As a cluster operator you may not need to use all these resources although you should be familiar with them to understand how the cluster is working and being used.

**Comment by steveperry-53** [comment]
3 Get information about my cluster -> 3 Get information about your cluster

Why do I think it should be "your"? The rest of the page has a "you" feel to it. And the first sentence in this section starts with "You can".

**Comment by steveperry-53** [comment]
access clusters using the cluster API -> access clusters using the Kubernetes API

**Comment by steveperry-53** [comment]
These two sentences don't work for me: "You can access clusters using the cluster API. If you are not already familiar, run through the tutorial to view your cluster’s Pods and Nodes."

If you are not already familiar with what? The Kubernetes API? I don't see how I can help my unfamiliarity with the Kubernetes API by doing this particular tutorial.

**Comment by steveperry-53** [comment]
There are three instances of "kubernetes". Need upper-case K.

**Comment by steveperry-53** [comment]
typically discuss simpler stateless applications -> typically discuss simple stateless applications

**Comment by steveperry-53** [comment]
As you move into fuller dev, testing, and production environments"

The word "fuller" sounds strange to me. Let's check with the others.

**Comment by steveperry-53** [comment]
These links should have title casing:
Managing Compute Resources for Containers
Configuration Best Practices

**Comment by steveperry-53** [comment]
Use sentence casing, not title casing for this heading: Work with ingress, networking, storage, workloads

Possibly capitalize Ingress because it's an API resource.

**Comment by steveperry-53** [comment]
Remove "TODO: identify major items".

**Comment by steveperry-53** [comment]
Don't say "currently".
Role-based access control (RBAC) is currently the recommended mechanism

**Comment by steveperry-53** [comment]
Comma after structure.

Kubernetes provides some basic logging structure and you may want to use additional tools to help aggregate and analyze log data.

**Comment by steveperry-53** [comment]
There is an overview at of tools at -> There is an overview of tools at

**Comment by steveperry-53** [comment]
several options to installing Prometheus -> several options for installing Prometheus

**Comment by steveperry-53** [comment]
and CoreOS provides a prometheus operator and kube-prometheus which adds on ->
and CoreOS provides prometheus-operator and kube-prometheus, which adds on

**Comment by steveperry-53** [comment]
Capitalize Minikube.

**Comment by steveperry-53** [comment]
This sentence has gotten off track: A common configuration on minikube and some kubernetes clusters used Heapster along with InfluxDB and Grafana.

Maybe this? A common configuration on Minikube and some Kubernetes clusters is to use Heapster along with InfluxDB and Grafana.

**Comment by heckj** [comment]
It got merged because @tengqm put an `/lgtm` On it, and by default approvers come in with “approved” already in place - so it had the tags it was looking for, and nothing else was stopping it (such as a `hold` tag). 

I love these edits, thank you - I’ll make another PR with the content of them as updates. It would have been fantastic if you could have provided them while it was still on a feature branch and prior to our merging of this content, but I’ll take it!

**Comment by heckj** [comment]
opened a full-on PR #7217 to address the additional feedback. Thanks Steve! Assigned you as reviewer and put a hold on the PR so that it won't merge prior to your full review

**Pull request merged**